ocis-pkg/log/log.go

```diff
@@ -132,10 +132,8 @@ func NewLogger(opts ...Option) Logger {
 		Timestamp().
 		Logger().Level(logLevel)
 
-	if logLevel == zerolog.DebugLevel {
-		var lineInfoHook LineInfoHook
-		logger = logger.Hook(lineInfoHook)
-	}
+	var lineInfoHook LineInfoHook
+	logger = logger.Hook(lineInfoHook)
```

Comment:
Hm, not sure if it is really a good idea to do this at all log level. Any idea how much overhead the runtime.Caller(3)` call adds? Doesn't it involve introspection (which IIRC would add quite a bit of overhead)?

Can we leave this at the Debug level for now?

Reply:
That was my request. I think it's valuable information. Can we log it on info at least? debug is usually so cluttered you can't find anything...

Reply:
I'll set it to info level then.

Reply:
wfm